build/handler/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM centos:7
+RUN yum -y install epel-release && \
+    yum -y update && \
+    yum -y install \
+        https://kojipkgs.fedoraproject.org//packages/nmstate/0.0.5/1.el7/noarch/python2-libnmstate-0.0.5-1.el7.noarch.rpm \
```

Comment:
// should probably be /

Reply:
Done, also upgraded nmstate to 0.0.6

Was entertaining the idea of using ubi7 or even ubi8 but looks like we need subscription-manager to install deps.